src/api/api.cc

```diff
@@ -11355,7 +11366,7 @@ void recordreplay::Assert(const char* format, ...) {
 }
 
 extern "C" void V8RecordReplayAssert(const char* format, ...) {
-  if (!gAssertsDisabled && recordreplay::IsRecordingOrReplaying()) {
+  if (recordreplay::HasAsserts()) {
```

Comment:
inconsistent about `HasAsserts()` vs `recordreplay::HasAsserts()` here

Reply:
This is not inconsistent. `V8RecordReplayAssert` is not in the `recordreplay` namespace, so it must have the prefix.